packages/editor/src/components/collab-sidebar/utils.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Sanitizes a comment string by removing non-printable ASCII characters.
+ *
+ * @param {string} str - The comment string to sanitize.
+ * @return {string} - The sanitized comment string.
+ */
+export function sanitizeCommentString( str ) {
+	return str.trim().replace( /[^\x20-\x7E]/g, '' );
+}
```

Comment:
Is this for removing binary characters? Don't we have a utility for this somewhere? Why is it necessary and hasn't been necessary so far?

Reply:
Why is this needed in the first place? I only see it being used for some disabled state.